charts/kube-prometheus-stack/README.md

```diff
@@ -1,6 +1,11 @@
 # kube-prometheus-stack
 
-Installs the [kube-prometheus stack](https://github.com/prometheus-operator/kube-prometheus), a collection of Kubernetes manifests, [Grafana](http://grafana.com/) dashboards, and [Prometheus rules](https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/) combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with [Prometheus](https://prometheus.io/) using the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator).
+This chart uses the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) to installs the [kube-prometheus stack](https://github.com/prometheus-operator/kube-prometheus), which is a collection of
```

Comment:
@jkroepke you mean this here is not sufficient for referencing to kube-prometheus? I only rephrased the content, it didn't actually change from what was there before...